content/applications/hr/referrals.rst

```diff
@@ -0,0 +1,703 @@
+instead of one job position at a time. To do this, navigate to the main Recruitment app dashboard.
+Click the :guilabel:`Email a friend` button at the bottom of the screen. A :guilabel:`Send Job Offer
+by Mail` pop-up window appears.
+
+Enter the email address(es) in the :guilabel:`Email` line. The email can be sent to multiple
+recipients, simply enter a comma followed by a single space after each email address. The
+:guilabel:`Subject` is pre-configured with `Job for you`, but can be edited.
+
+The email body is also populated with pre-configured text. The text that appears is:
+
+`Hello,`
+
+`There are some amazing job offers in my company! Have a look, they can be interesting for you`
+
+`See Job Offers`
+
+The `See Job Offers` text contains a tracking link to a complete list of all job positions currently
+being recruited for. Add any additional text and/or make any edits to the message body, then click
+:guilabel:`Send Mail` to send the email. The message will be sent and the window will close.
+
+.. _referrals/hired:
+
+Hired referrals
+===============
+
+When a candidate that has been referred by an employee is hired, the employee "grows their superhero
+team"  and adds superhero avatars to their Referrals dashboard.
+
+After a referral has been hired, when the employee next opens the Referrals app, instead of the main
+dashboard, a hired page loads. The text `(Referral Name) has been hired! Choose an avatar for your
+new friend!`
+
+Below this are five avatar thumbnails to choose from. If an avatar has already been assigned to a
+referral, the thumbnail is greyed out, and the name the avatar has been chosen for appears beneath
+the avatar. Click on an available avatar to select it.
+
+If more than one referral was hired since opening the Referrals application, after selecting the
+first avatar, the user is prompted to select anther avatar for the subsequent hired referral. Once
+all avatars have been selected, the dashboard loads and all the avatars are now visible. Mouse over
+each avatar and their name is displayed above them.
+
+.. image:: referrals/avatars.png
+   :align: center
+   :alt: The hired screen. A selection of avatars are presented to chose from, with any already
+         chosen are greyed out.
+
+Referral points
+===============
+
+The referral program is based on earning points that are then exchanged for rewards. On the main
+Referrals application dashboard, the top displays a summary of the user's current points. The left
+side of the summary displays the total points earned, and the right side displays the points that
+are available to spend.
+
+To see the various referrals and the points earned for each, click on one of the buttons directly
+beneath the avatars. The options are :guilabel:`Referrals`, :guilabel:`Ongoing`, and
+:guilabel:`Successful`.
+
+The number of referrals for each category is displayed above the word. The total number of referrals
```

Comment:
```suggestion
The number of referrals for each category is displayed above the title. The total number of referrals
```

Reply:
This phrasing is a bit odd ("number of referrals" vs "total number of referrals"). Its referring to two different numbers but using the same identifer?